Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -259,13 +278,15 @@ spec:
 
 ### Erasure Coding
 
-[Erasure coding](http://docs.ceph.com/docs/master/rados/operations/erasure-code/) allows you to keep your data safe while reducing the storage overhead. Instead of creating multiple replicas of the data,
-erasure coding divides the original data into chunks of equal size, then generates extra chunks of that same size for redundancy.
+[Erasure coding](http://docs.ceph.com/docs/master/rados/operations/erasure-code/) allows you to keep your data safe while reducing the storage overhead. Instead of creating multiple full replicas of the data,
+erasure coding (EC) divides the original data into chunks of equal size, then generates additional chunks of that size for redundancy.  This is akin to RAID5 or RAID6 but much more flexible.
 
-For example, if you have an object of size 2MB, the simplest erasure coding with two data chunks would divide the object into two chunks of size 1MB each (data chunks). One more chunk (coding chunk) of size 1MB will be generated. In total, 3MB will be stored in the cluster. The object will be able to suffer the loss of any one of the chunks and still be able to reconstruct the original object.
+For example, if you have a RADOS object of size 2MB, the simplest erasure coding with two data chunks would divide the object into two chunks of size 1MB each (data chunks). One more chunk (coding chunk) of size 1MB will be generated. In total, 3MB will be stored in the cluster. The object will be able to suffer the loss of any one of the chunks and still be able to reconstruct the original object.
```

Comment:
How can we best phrase this in terms of user data? Users don't have RADOS objects, just files. What about mentioning files instead for clarity?
```suggestion
For example, if you have a file of size 2MB, the simplest erasure coding with two data chunks would divide the file into two chunks of size 1MB each (data chunks). One more chunk (coding chunk) of size 1MB will be generated. In total, 3MB will be stored in the cluster. The file will be able to suffer the loss of any one of the chunks and still be able to reconstruct the original file.
```

Reply:
Strictly speaking Ceph doesn't see files so I didn't want to confuse CephFS, RBD, and RGW.  I put in language using an RGW object as an example instead, does that work for you?  I also recast this in terms of a 4+2 profile given the below blessing to retcon 2+1.

Reply:
RGW object sounds good, in addition to 4+2!